src/libraries/System.IO.Compression/src/System/IO/Compression/ZipHelper.cs

```diff
@@ -189,5 +190,15 @@ private static bool SeekBackwardsAndRead(Stream stream, byte[] buffer, out int b
                 return true;
             }
         }
+
+        internal static bool GetSuppressValidation()
+        {
+            if (AppContext.TryGetSwitch("System.Compression.SuppressValidation", out bool suppressValidation))
```

Comment:
Naming.  This should be `System.IO.Compression.ZipArchive.SuppressValidation` or  `System.IO.Compression.ZipArchiveEntry.SuppressValidation`.